### PR TITLE
Align code-block line numbers to the right

### PR DESCRIPTION
**Pull request by MSzabi**
## Type of Change

- **SCSS Styling:** Code block line number alignment.

## What issue does this relate to?
Aligns code block line numbers to the right.

### What should this PR do?
Aligns code block line numbers to the right.

### What are the acceptance criteria?
Code block line numbers are aligned to the right hand side.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ breaking changes to the options supported by the plugin will be shipped as a bre
 
 ## Unreleased changes
 
+- (minor) Align code-block line numbers to the right.
```
[the remark this answers is not in the document]
Comment (MSzabi) [reply CHANGELOG.md]: Changed.

File: styles/_code_prefix.scss [hunk 2]
```diff
@@ -50,7 +50,29 @@ pre {
 
                         &::before {
                             border-right: 1px solid rgba($white, 0.5);
+                            direction: rtl;
```
[the remark this answers is not in the document]
Comment (MSzabi) [reply styles/_code_prefix.scss]: The examples I found around this topic were with direction, but `text-align` seems to work just as fine.
Changed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ breaking changes to the options supported by the plugin will be shipped as a bre
 
 ## Unreleased changes
 
+- (patch) Align code-block line numbers to the right.
```
Comment (MattIPv4) [inline CHANGELOG.md:11]:
```suggestion
- (patch) Align code-block line numbers to the right
```
Nitpick, no period, and could this be placed after the comment (to reduce the diff for the next release)?

File: styles/_code_prefix.scss
[the remark this answers is not in the document]
Comment (MSzabi) [reply styles/_code_prefix.scss:1]: Done.

File: CHANGELOG.md (hunk 3 above)
Comment (MSzabi) [reply CHANGELOG.md]: Updated.

**Review by MattIPv4 — APPROVED** [review]

**Pull request merged**